exercises/react/example.py

```diff
@@ -0,0 +1,59 @@
+class Cell(object):
+    def __init__(self, reactor, value=0, dependencies=set(),
+                 updater=None):
+        self.reactor = reactor
+        self.index = len(reactor.cells)
+        reactor.cells.append(self)
+        self.value = value
+        self.dirty = False
+        self.dependencies = dependencies
+        self.dependents = set()
+        self.updater = updater
+        self.watchers = set()
```

Comment:
If you're expecting this to exist (which line 80 of the tests implies), you need to either document it or supply it in the template code. I think you need to revisit this exercise and work out what methods/attributes the tests require to exist and add them to the template or document their expected behaviour. Currently, someone would have to delve into the test-suite to work out what methods should exist.

Reply:
My original intent was to hide that functionality behind methods `add_watcher` and `remove_watcher`; I'm not sure why I did away with this, but I've restored those methods. `Reactor.watchers` is no longer needed by `react_test.py`